compiler/rustc_const_eval/src/const_eval/eval_queries.rs

```diff
@@ -243,6 +243,24 @@ pub(crate) fn turn_into_const_value<'tcx>(
     op_to_const(&ecx, &mplace.into(), /* for diagnostics */ false)
 }
 
+/// Computes the tag (if any) for a given type and variant.
+#[instrument(skip(tcx), level = "debug")]
+pub fn tag_for_variant_provider<'tcx>(
```

Comment:
This isn't really an "eval" query, so `mod.rs` may have been a better place... but overall, :shrug: not that important